File: Python3/ITP1_03_C.py
while True:
    n = [int(i) for i in input().split()]
    if n[0] == 0 and n[1] == 0:
        break
    if n[0] <= n[1]:
        print(str(n[0]) + " " + str(n[1]))
    else:
        print(str(n[1]) + " " + str(n[0]))

File: Ruby/ITP1_03_C.rb
while true
  n = gets.split(" ").map(&:to_i)
  if n[0] == 0 && n[1] == 0 then
    break
  end
  if n[0] < n[1] then
    puts "#{n[0]} #{n[1]}"
  else
    puts "#{n[1]} #{n[0]}"
  end
end